src/modules/keyboardmanager/common/Shortcut.cpp

```diff
@@ -395,28 +395,47 @@ void Shortcut::ResetKey(const DWORD& input, const bool& isWinBoth)
 // Function to return the string representation of the shortcut
 winrt::hstring Shortcut::ToHstring() const
 {
+    std::vector<winrt::hstring> keys;
+    GetKeyVector(keys);
+    
     winrt::hstring output;
+    for (auto& key : keys)
+    {
+        output = output + key + winrt::to_hstring(L" ");
+    }
+    if (keys.size() > 1)
+    {
+        return winrt::hstring(output.c_str(), output.size() - 1);   
+    }
+    else
+    {
+        return output;
+    }
+}
+
+size_t Shortcut::GetKeyVector(std::vector<winrt::hstring>& keys) const

```

Comment:
[nit] I'm not really sure what's the best convention to follow here. @ryanbodrug-microsoft is it better to just return the vector or pass the vector in an argument as reference and make the changes there, since the vector is an output of the function?

Reply:
I changed the signature to `std::vector<winrt::hstring> GetKeyVector()`.